assets/stylesheets/common/chat-browse.scss

```diff
@@ -91,4 +91,20 @@
       font-size: var(--font-up-1);
     }
   }
+
+  .chat-channel-card {
+    .chat-channel-card__leave-btn {
+      color: var(--tertiary-high);
```

Comment:
I think the colour can come from .btn-link

Reply:
it was slightly different in my test than the one used for members count

Reply:
and you're sure that isnt due to visited state?

Reply:
yes, I might be wrong, but I will merge like this and let you tweak if you see something wrong